content/concept-depositauth.md

```diff
@@ -0,0 +1,59 @@
+
+## Background
+
+Financial services regulations and licenses may require that a business or entity must know the sender of all transactions it receives. This presents a challenge on a decentralized system like the XRP Ledger where participants are identified by pseudonyms which can be freely generated and the default behavior is for any address to be able to pay any other.
+
+The Deposit Authorization flag introduces an option for those using the XRP Ledger to become compliant with such regulations without changing the fundamental nature of the decentralized ledger. With Deposit Authorization enabled, an account can only receive funds it explicitly approves by sending a transaction. The owner of an account using Deposit Authorization can perform the due diligence necessary to identify the sender of any funds _before_ sending the transaction that causes it to receive the money.
+
+Deposit Authorization is intended to be used with [Checks](reference-amendments.html#checks), [Escrow](concept-escrow.html), and [Payment Channels](reference-amendments.html#paychan). The "two-step" transaction model allows the sender to first authorize the sending of funds, then the destination to authorize the receipt of funds. Deposit Authorization cannot be used with [Payment transactions][] at present.
+
+## Recommended Usage
+
+To get the full effect of Deposit Authorization, Ripple recommends also doing the following:
+
+- Always maintain an XRP balance higher than the minimum [reserve requirement](concept-reserves.html).
+- Keep the DefaultRipple flag in its default (disabled) state. Do not enable [rippling](concept-noripple.html) on any trust lines.
+- Do not place [Offers](reference-transaction-format.html#offercreate). It is impossible to know in advance which matching offers will be consumed to execute such a trade.
+
+## Precise Semantics
+
+An account with Deposit Authorization enabled:
+
+- **Cannot** be the destination of [Payment transactions][], with **one exception**:
+    - If the account's XRP balance is equal to or below the minimum account [reserve requirement](concept-reserves.html), it can be the destination of a Payment whose `Amount` is equal or less than the minimum account reserve (currently 20 XRP). This is to prevent an account from becoming "stuck" by being unable to send transactions but also unable to receive XRP. The account's owner reserve does not matter for this case.
+- Can receive XRP from [PaymentChannelClaim transactions][] **only if** the sender of the PaymentChannelClaim transaction is the destination of the payment channel.
+- Can receive XRP from [EscrowFinish transactions][] **only if** the sender of the EscrowFinish transaction is the destination of the escrow.
+- **Can** receive XRP or issued currencies by sending a [CheckCash][] transaction. _(Requires the [Checks amendment](reference-amendments.html#checks).)_
+- **Can** receive XRP or issued currencies by sending [OfferCreate transactions][]. ***TODO: test this***
```

Comment:
Just making sure I understand:  We recommend _[against_ placing offers](https://github.com/ripple/ripple-dev-portal/pull/297/files#diff-bd9394bad9fc1b6f7e3a205c09bde9ccR21) but it's technically possible to do so?